ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -225,14 +337,23 @@ export class AptosClient {
     return response.data;
   }
 
-  /** Submits a signed transaction to the transaction endpoint that takes JSON payload. */
+  /**
+   * Submits a signed transaction to the transaction endpoint that takes JSON payload
+   * @param signedTxnRequest A transaction, signed by `signTransaction` method
+   * @returns Transaction that is accepted and submitted to mempool
+   */
   async submitTransaction(signedTxnRequest: Types.SubmitTransactionRequest): Promise<Types.PendingTransaction> {
     const response = await this.transactions.submitTransaction(signedTxnRequest);
     raiseForStatus(202, response, signedTxnRequest);
     return response.data;
   }
 
-  /** Submits a signed transaction to the the endpoint that takes BCS payload. */
+  /**
+   * Submits a signed transaction to the the endpoint that takes BCS payload
+   * @param signedTxn A BCS transaction representation. It is a hex-encoded string prefixed with `0x`
+   * and fulfilled with two hex digits per byte
```

Comment:
This is not accurate. Lets remove **It is a hex-encoded string prefixed with `0x` and fulfilled with two hex digits per byte**. The bytes are just BCS bytes, no hex-encoding. We need to update the openapi spec as well. I will do that.